assets/css/style.css

```diff
@@ -3,35 +3,64 @@
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
+  padding-bottom: 2rem;
+}
+
```

Comment:
- [ ] [OPTIONAL] The pages should look almost identical to the original design, outside the content differences, please use the exact same colors, for text, backgrounds, borders, etc, as the original design
   - Basic color swatch: ```#272A31```, ```#EC5242```, ```#D3D3D3```